resources/magento2/common.properties

```diff
@@ -27,3 +27,10 @@ common.notAvailable=N/A
 common.classInheritance=Inherit Class
 common.license.proprietary=Proprietary
 common.description=Description
+common.cli.class.name=Class Name
+common.cli.parent.directory=Parent Directory
```

Comment:
Let's use it like this `common.parentDirectory` as it may reused in some other components.